src/curated-corpus/helpers/helperFunctions.ts

```diff
@@ -159,6 +160,76 @@ export const formatFormLabel = (str: string): string => {
   );
 };
 
+const DEFAULT_SURFACE_LOCALE = 'en-US';
+
+/**
+ * Extracts locale from a scheduled surface GUID by parsing the last two segments.
+ * Expected GUID format: "NEW_TAB_EN_US" -> "en-US"
+ * Handles INTL regions: "NEW_TAB_DE_INTL" -> "de-INTL"
+ *
+ * @param guid - The scheduled surface GUID (e.g., "NEW_TAB_EN_US")
+ * @returns Locale string in language-region format (e.g., "en-US")
+ */
+const inferLocaleFromGuid = (guid: string): string => {
+  const segments = guid.split('_');
+  if (segments.length < 3) {
+    return DEFAULT_SURFACE_LOCALE;
+  }
+
+  // Extract language and region from the last two segments of the GUID
+  const language = segments[segments.length - 2]?.toLowerCase();
+  const region = segments[segments.length - 1];
+
+  // Special handling for international regions
+  if (region === 'INTL') {
+    return `${language}-INTL`;
+  }
```

Comment:
hmm, still curious at this `if` statement. isn't it essentially equal to the `return` statement below on line 189?